<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launchPage</class>
 <widget class="QWidget" name="launchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Launch</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>DataPP view</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTreeView" name="dataPPView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>357</width>
               <height>138</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Threshold:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="KDoubleNumInput" name="thresholdNumInput">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.000100000000000</double>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="sliderEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Analyzing politics</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="analyzingPoliticsComboBox">
                  <item>
                   <property name="text">
                    <string>SingleStep</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Iterative</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>break on
every iteration</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Select backend to use</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="backendComboBox">
                  <property name="currentIndex">
                   <number>1</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>Redland</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Virtuoso</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Main Nepomuk Storage</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="cleanExaminedCheckBox">
                <property name="text">
                 <string>Auto clean examined info</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="startButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="KTabWidget" name="selectTargets">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Enter  uri</string>
             </attribute>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>333</width>
                <height>23</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Nepomuk Uri</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="KLineEdit" name="uriLineEdit"/>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_selectResources">
             <attribute name="title">
              <string>Select resources</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="Nepomuk::Utils::SearchWidget" name="resWidget"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_selectCategories">
             <attribute name="title">
              <string>Select Categories</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0" colspan="3">
               <widget class="QListView" name="categoryView"/>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QCheckBox" name="checkBox_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>take only </string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="maxResInQueryNumInput">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Decisions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Decision list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DecisionCollectionWidget" name="decisionListWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="KTabWidget" name="ktabwidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="decisionViewTab">
          <attribute name="title">
           <string>Decision</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="DecisionWidget" name="decisionWidget"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="identificationPage">
          <attribute name="title">
           <string>Identification </string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="onlyMainCheckBox">
             <property name="toolTip">
              <string>If checked, then only information about main resources of the Decision will be displayed.</string>
             </property>
             <property name="text">
              <string>Only main resources</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="identificationTableWidget">
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column/>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Controls</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0" colspan="3">
           <widget class="QProgressBar" name="decisionProgressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="KPushButton" name="applyDecisionButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="KPushButton" name="identifyDecisionButton">
            <property name="text">
             <string>Identify</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KPushButton" name="identifyMainButton">
            <property name="text">
             <string>Identify main</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DecisionWidget</class>
   <extends>QWidget</extends>
   <header>webextractor/decisionwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Nepomuk::Utils::SearchWidget</class>
   <extends>QWidget</extends>
   <header>nepomuk/searchwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DecisionCollectionWidget</class>
   <extends>QListWidget</extends>
   <header location="global">webextractor/decisioncollectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
